Make calico MTU configurable in bootstrap config (#2677)
scality/metalk8s: docs/installation/advanced-guide.rst

Advanced guide
==============

.. _multiple CIDR network:

Multiple CIDRs network
----------------------

In the Bootstrap Configuration it's possible to provide several CIDRs for a
single network, it's needed when several nodes does not sit in the same
network.

.. code-block:: yaml

     networks:
       controlPlane:
         cidr:
           - 10.100.1.0/28
           - 10.200.1.0/28
       workloadPlane:
         cidr:
           - 10.100.2.0/28
           - 10.200.2.0/28

> This kind of deployment need good knowledge about networking, as all workload
> node need to be able to communicate with all workload node even those that does
> not sit in the same CIDRs.

Reviewer: ```suggestion
This kind of deployment needs good knowledge about networking, as each workload
node needs to be able to communicate with all others, even those in a different
workload CIDR.
```
